README.md

```diff
@@ -1,16 +1,204 @@
-# UCAN Invocation
+# UCAN Invocation Specification v0.1.0
 
 ## Editors
 
-* _
+* [Brooklyn Zelenka](https://github.com/expede/), [Fission](https://fission.codes/)
 
 ## Authors
 
-* _
+* [Brooklyn Zelenka](https://github.com/expede/), [Fission](https://fission.codes/)
+* [Irakli Gozalishvili](https://github.com/Gozala), [DAG House](https://dag.house/)
 
-# 0. Abstract
+## Depends On
+
+* [`ucan-ipld`](https://github.com/ucan-wg/ucan-ipld/)
+
+# 0 Abstract
 
 ## Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
 
+# 1 Introduction
+
+## Separation
+
+Why separate the UCAN from the invocation format? Surely the UCAN itself already contains all the infomation required.
+
+I would argue that this is not the case for a few reasons.
+
+1. Authority is not intent to execute. Authority transfer is first-class in UCAN, other actions are not
+2. Mixing the two levels means that invocation info will live with the token (as a proof) on subdelegations
+3. Other detail, such as scheduling
```

Comment:
This is where I feel I'm missing something. Why does scheduling needs to be part of invocation as opposed to just another capability you can invoke with some parameters to schedule ?

Would there be other things along the lines of scheduler that were not considered and need to be introduced after the fact ?

Reply:
It's all about a scheduler in the "thread scheduler" sense. "Do it now" or "do this `n` times" is qualitatively different from "here, you now hold these keys, you're allowed to use my car for your own purposes".

As I'm thinking this through, I wonder if maybe DAG House's use cases tend to be very event oriented, and a lot of Fission's are in the bucket of enabling agents to do things for their own purposes. For example: "you're allowed to write into this bucket" is quite different from "write the contents of this CAR into this bucket ASAP".